src/Enums/ticketTypes.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
`TicketTypes.php` to make the autoloader happy

Reply:
I'm renaming them locally, but it looks like Git isn't recognizing them as changes. Someone with more know-how may need to take care of the renaming side of things...